docs/workflow-controller-configmap.yaml

```
@@ -225,8 +225,8 @@ data:
     metricsTTL: "10m"
     # IgnoreErrors is a flag that instructs prometheus to ignore metric emission errors. Default is "false"
     ignoreErrors: false
-    # Use a self-signed cert for TLS, default false
-    secure: false
+    # Use a self-signed cert for TLS, default true
```

this doc also has versions listed, so we should continue to do that:
```suggestion
    # Use a self-signed cert for TLS
    # >= 3.6: default true
```